### PR TITLE
Helm add resource block linkerd-jaeger chart

### PR DESCRIPTION
**Pull request by yuriydzobak**
Subject

Problem
jaeger-injector deployment doesn't have resource block. 

Solution
Added into values and deployment resources block

### CODE REVIEW

File: jaeger/charts/linkerd-jaeger/values.yaml
Comment (alpeb) [inline jaeger/charts/linkerd-jaeger/values.yaml:196]: Can you please change the comments here, as they should refer to the webhook container, and not the jaeger container?

**Review by alpeb — COMMENTED** [review]
Thanks @yuriydzobak. I've added a comment about the comments. Also please take care of the [DCO](https://github.com/linkerd/linkerd2/blob/main/CONTRIBUTING.md#developer-certificate-of-origin) for the PR checks to pass.

**Review by alpeb — COMMENTED** [review]
Thanks for the quick changes :+1: 

CI is still failing because some test golden files still need to be updated. You can do this automatically with `go test ./... -update` and push the changes. Also it seems you're using `helm-docs` v1.5, while our CI is using v1.4. You can call the command with `bin/helm-docs` and CI should stop complaning :wink:

**Review by Pothulapati — APPROVED** [review]
LGTM! Once the golden files are updated, Happy to merge this PR

**Review by alpeb — APPROVED** [review]
:+1:

**Pull request merged**
